rdt-open-reader/src/main/java/io/ona/rdt/fragment/TestsProfileFragment.java

```diff
@@ -152,7 +152,7 @@ public synchronized void onParasiteProfileFetched(List<ParasiteProfileResult> pa
             return;
         }
         try {
-            ParasiteProfileResult parasiteProfileResult = parasiteProfileResults.get(0);
+            ParasiteProfileResult parasiteProfileResult = parasiteProfileResults.get(parasiteProfileResults.size() - 1);
```

Comment:
Possiblity of `ArrayIndexOutOfBoundsException`

Reply:
The code above this ensures that the list is not empty.